### PR TITLE
feat(timescaledb): create a TimescaleDB documentation

### LINKED ISSUE #1511: [PostgreSQL] Document the TimescaleDB extension

### LINKED ISSUE #1512: [TimescaleDB] Document how to downsample and limit the data retention
https://docs.timescale.com/timescaledb/latest/how-to-guides/user-defined-actions/example-downsample-and-compress/#downsample-and-compress

### PR DESCRIPTION
**Pull request by curzolapierre**
Contains:
- TimescaleDB extension introduction
- Downsampling
- Data retention

Fix #1511 
Fix #1512

### CODE REVIEW

File: _posts/databases/postgresql/TimescaleDB/2000-01-01-downsample.md
Comment (Soulou) [inline _posts/databases/postgresql/TimescaleDB/2000-01-01-downsample.md:22]:
```suggestion
So we can use a downsampling logic which will reduce the datas older than one week,
```

**Review by Soulou — CHANGES_REQUESTED** [review]

**Comment by Soulou** [comment]
`downsample` is a verb, not a noun ;-)

File: _posts/databases/postgresql/TimescaleDB/2000-01-01-downsample.md
Comment (EtienneM) [inline _posts/databases/postgresql/TimescaleDB/2000-01-01-downsample.md:120]: I don't know if it matters for Jekyll but I would write a lower case `SQL`.

File: _posts/databases/postgresql/TimescaleDB/2000-01-01-downsample.md
Comment (EtienneM) [inline _posts/databases/postgresql/TimescaleDB/2000-01-01-downsample.md:124]:
```suggestion
This script invokes the created procedure `downsample_conditions` to apply it for
```

File: _posts/databases/postgresql/TimescaleDB/2000-01-01-intro.md
Comment (EtienneM) [inline _posts/databases/postgresql/TimescaleDB/2000-01-01-intro.md:11]: By being an external reader, I don't understand what you are talking about with this "Community edition". It seems it comes out of nowhere. Maybe it's less odd to say that the list of TimescaleDB functions introduced by enabling TimescaleDB on a PostgreSQL database is here (https://docs.timescale.com/api/latest/). From this list, the functions labelled as `Community` are not available on Scalingo.

**Review by EtienneM — CHANGES_REQUESTED** [review]

File: _posts/databases/postgresql/timescaledb/2000-01-01-intro.md
Comment (EtienneM) [inline _posts/databases/postgresql/timescaledb/2000-01-01-intro.md:11]: I find it ugly to have the URL displayed, I prefer a link with a meaningful text
```suggestion
TimescaleDB is a PostgreSQL extension for the support of time-series data.
This extension adds several features and functions. You can find more information
on the [official documentation](https://docs.timescale.com/api/latest).
```

File: _posts/databases/postgresql/timescaledb/2000-01-01-intro.md
Comment (EtienneM) [inline _posts/databases/postgresql/timescaledb/2000-01-01-intro.md:20]: I don't like this paragraph. I find it too deceptive. I don't see what it adds, I would remove it
```suggestion
```

File: _posts/databases/postgresql/timescaledb/2000-01-01-intro.md
Comment (EtienneM) [inline _posts/databases/postgresql/timescaledb/2000-01-01-intro.md:16]: Maybe this introduction page should talk about the two other pages created, and link to them?

File: _posts/databases/postgresql/timescaledb/2000-01-01-downsampling.md
Comment (EtienneM) [inline _posts/databases/postgresql/timescaledb/2000-01-01-downsampling.md:3]: You don't need a nav if it's the same as the title:
```suggestion
```

File: _posts/databases/postgresql/timescaledb/2000-01-01-downsampling.md
Comment (EtienneM) [inline _posts/databases/postgresql/timescaledb/2000-01-01-downsampling.md:8]: Simplification suggestion:

```suggestion
The downsampling reduces the amount of data by simplifying the old values.
```

I don't like the word "simplifying". I don't understand what it means

File: _posts/databases/postgresql/timescaledb/2000-01-01-downsampling.md
Comment (EtienneM) [inline _posts/databases/postgresql/timescaledb/2000-01-01-downsampling.md:15]: data is still uncountable. Do you have a dictionary installed in the text editor you use to write doc? It should highlight that `datas` does not exist

```suggestion
value. This is a use case where a table is composed of raw data.
```

File: _posts/databases/postgresql/timescaledb/2000-01-01-downsampling.md
Comment (EtienneM) [inline _posts/databases/postgresql/timescaledb/2000-01-01-downsampling.md:17]: nitpick: I'm not sure about the purpose of the word `Now` here. From my point of view it does not add any information.

```suggestion
We want to decrease the database size. We don't need to have data by the
```

File: _posts/databases/postgresql/timescaledb/2000-01-01-downsampling.md
Comment (EtienneM) [inline _posts/databases/postgresql/timescaledb/2000-01-01-downsampling.md:20]: We rarely use the future tense in documentation pages:
```suggestion
So we can use a downsampling logic which reduces the data older than one week,
and only stores the average by hour.
```

File: _posts/databases/postgresql/timescaledb/2000-01-01-downsampling.md
Comment (EtienneM) [inline _posts/databases/postgresql/timescaledb/2000-01-01-downsampling.md:25]: Not sure about this one

```suggestion
For the following data:
```

File: _posts/databases/postgresql/timescaledb/2000-01-01-downsampling.md
Comment (EtienneM) [inline _posts/databases/postgresql/timescaledb/2000-01-01-downsampling.md:39]: I didn't know but the word `bellow` exists and means "mugir" or "beugler" xD

```suggestion
The procedure below contains the logic to downsample the table `conditions`
```

File: _posts/databases/postgresql/timescaledb/2000-01-01-downsampling.md
Comment (EtienneM) [inline _posts/databases/postgresql/timescaledb/2000-01-01-downsampling.md:12]: If it's based on the official doc, please make `TimescaleDB documentation` a link to the said page

```suggestion
Below an example of a logic of downsampling. It is based on TimescaleDB documentation.
```

File: _posts/databases/postgresql/timescaledb/2000-01-01-downsampling.md
Comment (EtienneM) [inline _posts/databases/postgresql/timescaledb/2000-01-01-downsampling.md:97]: nitpick: not sure `To do so` adds anything here. And it really sounds like a French translation to my hears. I might be wrong:
```suggestion
Add a `cron.json` file at the root of your application’s source code,
```

File: _posts/databases/postgresql/timescaledb/2000-01-01-data-retention.md
Comment (EtienneM) [inline _posts/databases/postgresql/timescaledb/2000-01-01-data-retention.md:3]: Duplicate of the title, not needed 
```suggestion
```

File: _posts/databases/postgresql/timescaledb/2000-01-01-data-retention.md
Comment (EtienneM) [inline _posts/databases/postgresql/timescaledb/2000-01-01-data-retention.md:12]:
```suggestion
Below an example of a retention policy which applies on __all__ hypertables.
```

File: _posts/databases/postgresql/timescaledb/2000-01-01-data-retention.md
Comment (EtienneM) [inline _posts/databases/postgresql/timescaledb/2000-01-01-data-retention.md:14]: I don't understand this sentence

File: _posts/databases/postgresql/timescaledb/2000-01-01-data-retention.md
Comment (EtienneM) [inline _posts/databases/postgresql/timescaledb/2000-01-01-data-retention.md:39]:
```suggestion
You can schedule the execution of the previous procedure by using the
```

File: _posts/databases/postgresql/timescaledb/2000-01-01-downsampling.md
Comment (EtienneM) [inline _posts/databases/postgresql/timescaledb/2000-01-01-downsampling.md:95]:
```suggestion
You can schedule the execution of the previous procedure by using the
```

File: _posts/databases/postgresql/timescaledb/2000-01-01-data-retention.md
Comment (EtienneM) [inline _posts/databases/postgresql/timescaledb/2000-01-01-data-retention.md:56]:
```suggestion
command that invokes a SQL script called `data_retention_procedure.sql`.
```

**Review by EtienneM — CHANGES_REQUESTED** [review]
Thanks a lot for these pages. I didn't expect it to be that simple. I'm happy \o/

**Review by EtienneM — APPROVED** [review]

**Comment by EtienneM** [comment]
@curzolapierre we don't want auto merge on this PR. We don't want our documentation to talk about TimescaleDB whereas the product is not ready yet. After Léo approves this PR, we should wait for the end of the development of this product to merge the PR

**Review by Soulou — APPROVED** [review]
LGTM to merge after product release.

**Review by EtienneM — APPROVED** [review]

**Pull request merged**
